constraint/pkg/client/regolib/src.go

```diff
@@ -4,6 +4,17 @@ const (
 	targetLibSrc = `
 package hooks["{{.Target}}"]
 
+violation[response] {
+	data.hooks["{{.Target}}"].library.autoreject_review[[rejection, constraint]]
```

Comment:
`constraint` should be a property of `rejection`. This way handlers that reject for reasons that have nothing to do with the constraint don't need to worry about creating a dummy constraint.